src/oop/programs_with_multiple_classes/hard/q4/Bookcase.java

```diff
@@ -0,0 +1,11 @@
+package oop.programs_with_multiple_classes.hard.q4;
+
+public class Bookcase {
+    private int sizeCapacity;
+    private Book b1;
```

Comment:
I have to double-check if we can have this kind of object relation in this category. I will ask Bowen and get back to you, but just leave it the way it is for now.

Reply:
Did she get back to you?